app/services/description_generator.rb

```diff
@@ -21,7 +21,7 @@ def generate
       title: title,
       # TODO: keywords not yet in model.
       note: [abstract, citation, contact],
-      event: [created_date]
+      event: [created_date].compact
```

Comment:
I guess this is more or less the same as `Array(created_date)`? I think it's a six-of-one-half-dozen-of-the-other situation, though.

Reply:
no, this eliminates `[nil]`, which `Array(created_date)` would not.